components/cli/pkg/util/helm.go

```diff
@@ -0,0 +1,180 @@
+		if !strings.Contains(v.Name, "NOTES.txt") {
+			tmplString = tmplString + "\n---\n" + v.Content
+		}
+	}
+	return tmplString
+}
+
+//Read helm chart from the filesystem.
+func GetChartFromDir(dir string) (*chart.Chart, error) {
+	helmChart, err := chartutil.LoadDir(dir)
+	if err != nil {
+		return nil, errors.Wrapf(err, "loading chart templates.")
+	}
+	return helmChart, err
+}
+
+//Read helm chart value file.
+//Chart path differs based on the installed OS.
+func GetHelmChartsCustomValues(chartName string, chartPath string, valueFile string) (string, error) {
+	data, err := ioutil.ReadFile(filepath.Join(chartPath, chartName, valueFile))
+	if err != nil {
+		log.Printf("Value file access error %s \n", err)
+		return "", err
+	}
+	return string(data), nil
+}
+
+func GetHelmChartDefaultValues(chartName string) (string, error) {
+	data, err := ioutil.ReadFile(filepath.Join(filepath.Join(CelleryInstallationDir(), constants.HelmCarts), chartName, "values.yaml"))
+	if err != nil {
+		log.Printf("Value file access error %s \n", err)
+		return "", err
+	}
+	return string(data), nil
+}
+
+//Apply generated k8s artifacts.
+func ApplyHelmTemplates(tmplString string) error {
+	cmd := exec.Command("kubectl", "apply", "-f", "-")
+	stdin, err := cmd.StdinPipe()
+	if err != nil {
+		log.Printf("kubectl command build error %s \n", err)
+	}
+	go func() {
+		defer stdin.Close()
+		io.WriteString(stdin, tmplString)
+	}()
+	out, err := cmd.CombinedOutput()
+	log.Printf("kubectl apply commnd output : %s", string(out))
+
+	if err != nil {
+		log.Printf("kubectl command execution error %s \n", err)
+		return err
+	}
+	return nil
+}
+
+func ApplyK8sResource(operation string, tmplString string, namespace string) error {
+	cmd := exec.Command("kubectl", operation, "-f", "-", "-n", namespace)
+	stdin, err := cmd.StdinPipe()
+	if err != nil {
+		log.Fatal("kubectl command build", err)
+	}
+	go func() {
+		defer stdin.Close()
+		io.WriteString(stdin, tmplString)
+	}()
+	out, err := cmd.CombinedOutput()
+	log.Printf("kubectl apply commnd output : %s", string(out))
+	if err != nil {
+		log.Print("kubectl command execution", err)
+		return err
+	}
+	return nil
+}
+
+//Apply helm chart with default values
+func ApplyHelmChartWithDefaultValues(chartName string, namespace string) error {
+	chartTemplate := RenderHelmChart(chartName, namespace, filepath.Join(UserHomeCelleryDir(), constants.HelmCarts, chartName), "")
+	err := ApplyHelmTemplates(chartTemplate)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func ApplyHelmChartWithDefaultValuesCustomCmd(chartName string, namespace string, operation string) error {
+	chartTemplate := RenderHelmChart(chartName, namespace, filepath.Join(UserHomeCelleryDir(), constants.HelmCarts, chartName), "")
+	err := ApplyK8sResource(operation, chartTemplate, namespace)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func ApplyHelmChartWithCustomValues(chartName string, namespace string, operation string, values string) error {
```

Comment:
IMHO, these functions are related to the runtime and hence should be moved to runtime package. However, this is not an urgent requirement.